pkg/istio/rbac/v1/provider_test.go

```diff
@@ -868,3 +874,158 @@ func TestGetCurrentIstioRbac(t *testing.T) {
 		})
 	}
 }
+
+const (
+	DomainName = "test.namespace"
+	username   = "user.name"
+	username1  = "user.*"
+)
+
+var (
+	onboardedService = &k8sv1.Service{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "onboarded-service",
+			Namespace: "test-namespace",
+			Annotations: map[string]string{
+				"authz.istio.io/enabled": "true",
+			},
+			Labels: map[string]string{
+				"app": "productpage",
+			},
+		},
+	}
+)
+
+func TestConvertAthenzModelIntoIstioAuthzPolicy(t *testing.T) {
```

Comment:
We should test the same functionality we're currently testing in the service role / service role binding tests. I'm okay if we make this in a separate PR.